test/regression/simple-linear-regression.js

```diff
@@ -49,5 +51,17 @@ describe('Simple linear regression', function () {
         regression.computeX(5).should.equal(2.5);
         regression.computeX(9).should.equal(0.5);
         regression.computeX(-12).should.equal(11);
+
+        regression.toString(3).should.equal('y = -2.00x + 10.0');
+    });
+    it('SLR constant', function () {
```

Comment:
it's because they don't have anything special. they are just 3 different regressions, 2 taken from the web and one that I invented